Makefile.common

```diff
@@ -1,8 +1,20 @@
-REPO=enmasseproject/$(PROJECT_NAME)
-DOCKERFILE_DIR ?= .
-COMMIT ?= latest
-
-all: build
-	echo "Running docker build"
-	curl -s https://raw.githubusercontent.com/EnMasseProject/travis-scripts/master/docker-build.sh | bash /dev/stdin $(REPO) $(DOCKERFILE_DIR)
-	curl -s https://raw.githubusercontent.com/EnMasseProject/travis-scripts/master/docker-push.sh | bash /dev/stdin $(REPO)
+DOCKERFILE_DIR  ?= ./
+DOCKER_REGISTRY ?= docker.io
+DOCKER_ORG      ?= $(USER)
+PROJECT_NAME    ?= $(shell basename $(CURDIR))
```

Comment:
What does the `shell` do?

Reply:
Good catch. That is a copy paste code which anyway doesn't work for us (since for example `kafka-connect-s2i` image is in directory `s2i`)